From d4a4fb28ea0c057428ef1f28bc689b8d0f085dc6 Mon Sep 17 00:00:00 2001
From: Frediano Ziglio <fziglio@redhat.com>
Date: Mon, 28 May 2018 09:20:41 +0100
Subject: [PATCH 12/43] msi: Do not generate deps.txt

There's no reason to tell the package installed on the build system
used.

Signed-off-by: Frediano Ziglio <fziglio@redhat.com>
Acked-by: Christophe Fergeau <cfergeau@redhat.com>
---
 Makefile.am          | 5 +----
 spice-vdagent.wxs.in | 4 ----
 2 files changed, 1 insertion(+), 8 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 62640f2..3020824 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -100,15 +100,12 @@ test_log_win_SOURCES =			\
 	common/test-log.cpp		\
 	$(NULL)
 
-deps.txt:
-	$(AM_V_GEN)rpm -qa | grep $(host_os) | sort | unix2dos > $@
-
 MANUFACTURER = The Spice Project
 
 EXTRA_DIST += spice-vdagent.wxs.in
 CONFIG_STATUS_DEPENDENCIES = spice-vdagent.wxs.in
 
-spice-vdagent-$(WIXL_ARCH)-$(VERSION)$(BUILDID).msi: spice-vdagent.wxs deps.txt all
+spice-vdagent-$(WIXL_ARCH)-$(VERSION)$(BUILDID).msi: spice-vdagent.wxs all
 	$(AM_V_GEN)DESTDIR=`mktemp -d`&&				\
 	make -C $(top_builddir) install DESTDIR=$$DESTDIR >/dev/null &&	\
 	MANUFACTURER="$(MANUFACTURER)" wixl -D SourceDir=$(prefix)	\
diff --git a/spice-vdagent.wxs.in b/spice-vdagent.wxs.in
index 452f995..7432ca9 100644
--- a/spice-vdagent.wxs.in
+++ b/spice-vdagent.wxs.in
@@ -61,9 +61,6 @@
                             Wait="yes"/>
             </Component>
           </Directory>
-          <Component Id="CDepsTxt" Guid="*">
-            <File Id='depstxt' Name='deps.txt' DiskId='1' Source='deps.txt' KeyPath='yes'/>
-          </Component>
         </Directory>
       </Directory>
     </Directory>
@@ -71,7 +68,6 @@
     <Feature Id="Complete"  Level="1">
       <ComponentRef Id="CSpiceAgent"/>
       <ComponentRef Id="CSpiceService"/>
-      <ComponentRef Id="CDepsTxt"/>
     </Feature>
 
   </Product>
-- 
2.17.1

